src/Components/InstanceTypes/SuggestedInstanceTypesTable.js

```diff
@@ -23,6 +23,8 @@ export default function SuggestedInstanceTypesTable() {
     const dispatch = useDispatch();
     const [page, setPage] = useState(PAGE);
     const [perPage, setPerPage] = useState(PER_PAGE);
+    const [activeSortIndex, setActiveSortIndex] = useState(3);
```

Comment:
@r14chandra rather than `activeSortIndex` state, we can have `activeSortKey` state here if we decide to change the suggested instance array format.

Reply:
I think both will work. You can keep existing state and just find key/column value based on index before calling dispatch at [line 37](https://github.com/RedHatInsights/ros-frontend/pull/292/files#diff-50cdac51d9b8f5bf46b02189c3f7fe47522e8f9f0928f5f40f6f2b7af6b83009R37).